lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -66,9 +66,21 @@ def filter_graph
         end
 
         def map_results
-          results_mapper_service.map_values(graph: @filtered_graph,
-                                            subject_uri: uri,
-                                            predicate_map: preds_for_term)
+          predicate_map = preds_for_term
+          ldpath_map = ldpaths_for_term
```

Comment:
Supports both ldpath_map (preferred) and predicate_map (deprecated).  Cannot mix ldpath and predicates.  A configuration must define results using one or the other.